### PR TITLE
Milestone 7: Add forms

### PR DESCRIPTION
**Pull request by BenMKT**
## Project implementations:

- Added forms to project application.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -125,9 +28,25 @@
...
+        <div class="actions">
+          <%= form.submit 'Create Post' %>
+        </div>
+      <% end %>
+
       <div class="see-all-posts">
-        <%= link_to "View all posts", user_posts_path(@user) %>
+        <%= link_to "See all posts", user_posts_path(@user) %>
       </div>
     </div>
```
Comment (Qoosim) [inline app/views/users/show.html.erb]: - [x] Great job adding forms to your app. However, I am unable to see the list of posts created by a user when I did so. A user should be able to see list of posts when clicking on the `See all posts` link. This feature is not functioning as expected at this moment. Kindly look into it and fix.

### App Behaviour

[post-error.webm](https://github.com/BenMKT/BlogApp/assets/26597432/d9ad1a88-fb9e-401d-8242-fc1d14dc8b30)

- [x] As a result of my inability to see the user posts, I cannot go further to test for the comments for a post and the likes for a post. Kindly check these two requirements too before asking for a new review. Good luck.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @BenMKT, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Implemented some project requirements. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (BenMKT) [reply app/views/users/show.html.erb]: - Refactored code base to resolve bugs.

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @BenMKT 

Good job implementing the suggested changes💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Implemented suggested changes
-  ✅ There is a form to create a post
-  ✅ There is a form to create a comment
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**